ipv-stub/src/endpoints/ipv-authorize.ts

```diff
@@ -71,3 +75,24 @@ async function get(
     );
   }
 }
+
+async function post(
+  _event: APIGatewayProxyEvent
+): Promise<APIGatewayProxyResult> {
+  const redirectUri = `${ROOT_URI}/ipv-callback`;
```

Comment:
This won't currently work as /ipv-callback in the frontend is the route for returning from the prove identity journey. I'm not sure that we do this journey any more post orch split so may be that we can repurpose, but currently in draft [PR](https://github.com/govuk-one-login/authentication-frontend/pull/2309) it's /auth-ipv-callback